Using device: cuda
Epoch 1/50
  0%|                                                                                                                             | 0/15 [00:00<?, ?it/s]/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/functional.py:756: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
  0%|                                                                                                                             | 0/15 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "train.py", line 191, in <module>
    train(net, args)
  File "train.py", line 67, in train
    output = net(t1_img, t2_img)
  File "/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home_ext/whm_ext/hcq/7/flood_unet/network/model.py", line 52, in forward
    s2_feature = self.s2_stream(t2_img)
  File "/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home_ext/whm_ext/hcq/7/flood_unet/network/model.py", line 115, in forward
    x1 = self.inc(x)
  File "/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home_ext/whm_ext/hcq/7/flood_unet/network/model.py", line 161, in forward
    x = self.conv(x)
  File "/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home_ext/whm_ext/hcq/7/flood_unet/network/model.py", line 151, in forward
    x = self.conv(x)
  File "/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/modules/container.py", line 139, in forward
    input = module(input)
  File "/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/modules/activation.py", line 102, in forward
    return F.relu(input, inplace=self.inplace)
  File "/home_ext/whm_ext/miniconda3/envs/camera/lib/python3.8/site-packages/torch/nn/functional.py", line 1452, in relu
    result = torch.relu(input)
RuntimeError: CUDA out of memory. Tried to allocate 512.00 MiB (GPU 0; 23.68 GiB total capacity; 14.68 GiB already allocated; 216.06 MiB free; 15.12 GiB reserved in total by PyTorch)